client/src/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const HOSTNAME = '127.0.0.1:3001';
-// export const HOSTNAME = '192.168.99.100:3001';
+// export const HOSTNAME = '127.0.0.1:3001';
+export const HOSTNAME = '192.168.99.100:3001';
```

Comment:
if you are not me and testing this you probably want to change it back to 127.0.0.1:3001